core/src/scene/styleParam.cpp

```diff
@@ -456,9 +464,18 @@ bool StyleParam::isColor(StyleParamKey _key) {
 bool StyleParam::isWidth(StyleParamKey _key) {
     switch (_key) {
         case StyleParamKey::width:
-        case StyleParamKey::font_stroke_width:
         case StyleParamKey::outline_width:
         case StyleParamKey::size:
+        case StyleParamKey::font_stroke_width:
+            return true;
+        default:
+            return false;
+    }
+}
+
+bool StyleParam::is2DWidth(StyleParamKey _key) {
```

Comment:
I would prefer Offset(s), just like the style param name, if I'm not missing something.